index.js

```diff
@@ -332,7 +332,7 @@ function cms(loopbackApplication, options) {
 
   if (config.allowUnsafeUpsert) {
```

Comment:
Other projects have been using it in the current location. Also, we probably don't want to expose that to the client side so explicitly.